app/models/study.rb

```diff
@@ -256,6 +256,7 @@ class Study < ApplicationRecord # rubocop:todo Metrics/ClassLength
     # External Customers
     custom_attribute(:s3_email_list)
     custom_attribute(:data_deletion_period)
+    custom_attribute(:contaminated_human_data_access_group)
```

Comment:
Could you add some tests for this?
(Update study_metadata factory or add a variation or trait for this if necessary)
Add test in `spec/models/study_spec.rb` to check it is included in `study.study_metadata`?
Also test that it defaults to NULL as in acceptance criteria.

Reply:
I am trying to add some code, but Codecov is already complaining about it.